tests/system/test_instrumentation.py

```diff
@@ -15,6 +15,8 @@
 os.environ["ELASTIC_APM_API_REQUEST_TIME"] = "1s"
 
 
+# Exercises the DEPRECATED apm-server.instrumentation.* config
+
```

Comment:
The tests in `test_libbeat_instrumentation` are copies of this file. I assume you didn't DRY the tests based on the fact that the support for `apm-server.instrumentation`  will be removed in 8.0.?
Please add a comment in both, to make future changes to the test cases in both files, and to delete this file for 8.0.